source/MicroBitPowerManager.cpp

```diff
@@ -475,10 +475,10 @@ void MicroBitPowerManager::deepSleep()
         return;
     }
 
-    CODAL_TIMESTAMP eventTime = 0;
-    bool wakeOnTime = system_timer_deepsleep_wakeup_time( eventTime);
-    int can = canDeepSleep( wakeOnTime, eventTime, true /*wakeUpSources*/, NULL /*wakeUpPin*/);
-    if ( can == DEVICE_OK)
+    //CODAL_TIMESTAMP eventTime = 0;
+    //bool wakeOnTime = system_timer_deepsleep_wakeup_time( eventTime);
+    //int can = canDeepSleep( false, eventTime, true /*wakeUpSources*/, NULL /*wakeUpPin*/);
+    //if ( can == DEVICE_OK)
```

Comment:
Can we just delete these lines rather than leave dead code?

Reply:
Sure, I mostly just left those in as a prompt for me for the old behaviour

Reply:
Removed: https://github.com/lancaster-university/codal-microbit-v2/pull/229/commits/610bafb4c4c049338babbce6fde4fa4bdcc3d972